htdocs/fourn/facture/card.php

```diff
@@ -929,7 +929,7 @@
 		}
 
 		// Standard invoice or Deposit invoice, not from a Predefined template invoice
-		if (GETPOST('type') == FactureFournisseur::TYPE_STANDARD || GETPOST('type') == FactureFournisseur::TYPE_DEPOSIT && GETPOST('fac_rec') <= 0) {
```

Comment:
On this line I didn't just reorder the conditions but I added missing parentheses